hack/deploy.sh

```diff
@@ -72,7 +72,7 @@ for namespace in ${namespaces[@]}; do
     fi
 done
 
-if [ "${CMD}" == "oc" ]; then
+if [ "${IS_OPENSHIFT}" == "true" ]; then
     # Switch project to kubevirt-hyperconverged
     oc project "${HCO_NAMESPACE}"
```

Comment:
Basically instead `oc` we can use here `${CMD}` (which can be either kubectl or oc)
but for now we are good, it just mean `oc` binary should exists for OpenShift clusters